### PR TITLE
Docking mechanic + Star Trek theming

### PR DESCRIPTION
**Pull request by Kmc2000**
Somewhat inspired by:
https://tgstation13.org/phpBB/viewtopic.php?f=9&t=18948&p=430524&hilit=gamemode+bounty#p430524

Ships will now dock with the station. These can either be positive encounters or negative ones. I have not yet added the customs system where you can reject ships as this is an alpha.

Procedure:
Announcement made, players are offered spawns on the ships.
Security scans the ship with the transporter and beams down people to search it if needs be.
If all is well, security will beam up the crews from the ships and they can join the round.

In total, there are 4 ship spawns per round at 3-10 minute intervals (random). Currently this encompasses every spawn for testing purposes.
There are 11 player spawns available within these 4

Other things:

Tazers are gone. They've been replaced with phasers and custom special FX. The HOS gets a phaser rifle which is super strong.
Klingons are now in the game! (Not roundstart choose-able yet)
aiannounce.ogg has been changed into a star trek themed version to avoid mass file changes.
Removed the buggy pockets from trek jackets

Feedback and event ideas welcome

:cl: Kmc2000
add: Ships will now dock with the station! This can either be a really good thing or a really bad thing.
/:cl:

### CODE REVIEW

**Comment by spookydonut** [comment]
Atomize dis

**Comment by Kmc2000** [comment]
This PR needs all of these components, especially the species

**Review by Jalleo — CHANGES_REQUESTED** [review]
PR the projectile and all of that separately to quickly merge that sooner than this. I want to properly review the docking mechanic code but I don't want to delay this other stuff which is suitable to add in sooner.

**Comment by Jalleo** [comment]
I also fixed your spelling errors in the PR text.

File: DS13/code/modules/borg/conversion.dm [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+	to_chat(M, "<span class='warning'>[user] pierces you with their assimilation tubules!</span>")
+	SEND_SIGNAL(M, COMSIG_LIVING_MINOR_SHOCK)
+	M.Jitter(3)
+	M.visible_message("<span class='warning'>[user] pierces [M] with their assimilation tubules!</span>")
+	playsound(M.loc, 'sound/weapons/pierce.ogg', 100,1)
+	if(do_after(user, 50, target = M)) //5 seconds
+		M.make_borg()
+		return
+
+/obj/item/borg_tool/afterattack(atom/I, mob/living/user, proximity)
```
Comment (Kmc2000) [inline DS13/code/modules/borg/conversion.dm:181]: Yeah what the shit is this I really need to fix this huh

File: DS13/code/modules/borg/conversion.dm [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+	var/sound = 'DS13/sound/effects/borg/machines/alcove.ogg'
+	var/mob/living/carbon/human/user
+	var/charge_time = 0
+	var/charge_amount = 10 //10 per tick, with a max charge storage of 1000
+
+/obj/structure/chair/borg/charging/New()
+	. = ..()
+	START_PROCESSING(SSobj, src)
+
+/obj/structure/chair/borg/charging/process()
```
Comment (Kmc2000) [inline DS13/code/modules/borg/conversion.dm:322]: This also sucks

**Comment by Kmc2000** [comment]
This will need a mapmerge on CentCom.dmm

**Comment by Jalleo** [comment]
This PR is Do not merge until the borg PR has been fully reviewed and merged.

**Pull request closed without merging**